tests/components/homeassistant_sky_connect/test_init.py

```diff
@@ -258,16 +272,24 @@ async def test_setup_entry_wait_usb(hass: HomeAssistant) -> None:
         "homeassistant.components.homeassistant_sky_connect.usb.async_is_plugged_in",
         return_value=False,
     ) as mock_is_plugged_in:
-        assert not await hass.config_entries.async_setup(config_entry.entry_id)
+        task = hass.async_create_task(
+            hass.config_entries.async_setup(config_entry.entry_id)
+        )
+        await asyncio.sleep(0.1)
```

Comment:
This sleep will be removed and replaced with something better if the solution is deemed OK.